controllingChargebyteBoard.py

```diff
@@ -0,0 +1,94 @@
+import socket
+
+
+
+class controllingChargebyteBoard:
+
+
+    def __init__( self, host: str, port: int ):
+        self.s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        self.s.connect((host, port))
+
+
+    def send_packet( self, service_id: int, payload: bytearray ):
+        self.s.send(bytearray([service_id] + payload))
+
+
```

Comment:
i needed to put on different functions, because they return completely different things. one returns the software and hardware number and the other returns the 'build', (whatever that means).